tools/download_wiki_index.sh

```diff
@@ -1,11 +1,51 @@
 #!/bin/bash
 
-if [ ! -d wiki_indri_index ];
-then
-  wget http://web.eecs.umich.edu/~jahausw/download/wiki_indri_index.tar.gz
+if [ -d wiki_indri_index ]; then
+  echo "Wikipedia index already downloaded, skipping"
+  exit 1
+else
+  echo "Wikipedia index not found: OK"
+fi
+
+FREE=`df -k --output=avail "$PWD" | tail -n1`
+if [[ $FREE -lt 11523524 ]]; then
+     echo "Enough space to download.: FAIL"
+     read -p "Not enough space. Do you want to continue (Y/n)? " -n 1 -r
+     if [[ ! $REPLY =~ ^[Yy]$ ]]
+     then
+        echo ""
+        echo "Aborting."
+        exit 1
+     fi
+    echo ""
+else
+   echo "Enough space to download.: OK"
+fi;
+
+echo "Starting download."
+
+wget -c http://web.eecs.umich.edu/~jahausw/download/wiki_indri_index.tar.gz
```

Comment:
Host this? As in move it from somewhere other than "http://web.eecs.umich.edu/~jahausw/download/wiki_indri_index.tar.gz"?